### ISSUE TITLE
Visualization for slightly smaller images?

### ISSUE THREAD

**Issue by shenglih** [lmzintgraf/DeepVis-PredDiff#7]
Excuse my naive question... but would you mind letting me know why images slightly smaller such as 200X200 were excluded for your method? Would it work on such images? If not, why wouldn't and what are the alternatives? Thanks a lot in advance for your guidance.

**Comment by lmzintgraf** [comment]
Yes, the method works on smaller images as well. In fact, any other data format besides from what I have in my example could would work, for example vectors of any length, or 3D images (as shown in the paper with the MRI data). You will have to adapt the code for this, since the current implementation is for images of size from 227x227 (this is what the pretrained models for Imagenet that I used were implemented for; hence the choice of that size).

**Comment by shenglih** [comment]
Thanks a lot Luisa!
